Makefile

```diff
@@ -218,23 +234,23 @@ generate-windows-icon:
 
 assets: $(RESOURCES_DOT_GO)
 
-linux-386: $(RESOURCES_DOT_GO)
+linux-386: $(RESOURCES_DOT_GO) $(SOURCES)
```

Comment:
By depending on $(SOURCES), we get this and similar targets to automatically rebuild if any of the go source files changed.